src/components/Category/ThemeCard.tsx

```diff
@@ -0,0 +1,41 @@
+import type { GiftThemeType } from '@/data/categoryMockData';
+import styled from '@emotion/styled';
+
+const ThemeCardContainer = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: column;
+
+  padding-top: ${({ theme }) => theme.spacing[0]};
+  padding-bottom: ${({ theme }) => theme.spacing[2]};
+  box-sizing: border-box;
+
+  width: ${({ theme }) => theme.components.category.themeCardWidth};
+  height: ${({ theme }) => theme.components.category.themeCardHeight};
+
+  cursor: pointer;
+`;
+
+const ThemeImage = styled.img`
+  width: ${({ theme }) => theme.components.category.themeCardImageSize};
+  height: ${({ theme }) => theme.components.category.themeCardImageSize};
+`;
+
+const ThemeName = styled.p`
+  font-size: ${({ theme }) => theme.typography.label.label2Regular.fontSize};
+  font-weight: ${({ theme }) => theme.typography.label.label2Regular.fontWeight};
+`;
+
+function ThemeCard({ theme }: { theme: GiftThemeType }) {
+  const { name, image } = theme;
```

Comment:
theme라는 객체가 아니라 name, image 라는 값으로 받아오면 어떨까요?


```suggestion
function ThemeCard({ name, image }: GiftThemeType) {
```

이렇게 했을 때 렌더링 관점에서 어떤 차이가 있을지 고민해보시면 좋겠어요!